docs/compression_algorithms/Quantization.md

```diff
@@ -230,6 +231,56 @@ file.
 
 ---
 
+#### AutoQ
+Different model architectures possess varying redundancy and sensitivity towards accuracy. Different device architecture may provide different performance capabilities. A mixed-precision model performs efficiently on a device may behave differently on another. It is a scaling challenge to find respective optimal precision configuration for a range of platforms (CPU, GPU, VPU etc and SKUs). To this end, NNCF provides an alternate mode, namely AutoQ, for mixed-precision automation. Based on HAQ, it employs deep reinforcement learning algorithm to automate the explorations and eventually learn the quantization policy that achieves optimal trade-off between accuracy, performance and user-specified constraints (e.g. model size, latency, compute complexity, power). 
```

Comment:
It looks like good stuff for the blogpost ) But can we make it shorter in the documentation? Like,
"AutoQ provides an alternate mode for mixed-precision quantization that utilizes an actor-critic algorithm, Deep Deterministic Policy Gradient (DDPG) for efficient search over the bitwidth space..."
It is probably worth pointing to the paper where the original implementation is disclosed.

Reply:
Condensed first two paragraphs into one. ) The remaining paragraphs contain algorithm description, keeping them for the documentation. 

Yes, that was a miss. Link to HAQ is embedded now.